src/views/become-a-scratcher/become-a-scratcher.jsx

```diff
@@ -0,0 +1,584 @@
+];
+
+const OnboardingHeader = ({user, sectionText, secondary}) => {
+    const [showModal, setShowModal] = useState(false);
+
+    return (
+        <div className="header">
+            {/* Finish Later Modal */}
+            <Modal
+                isOpen={showModal}
+                showCloseButton
+                useStandardSizes
+                className="mod-join"
+                shouldCloseOnOverlayClick={false}
+                onRequestClose={() => setShowModal(false)}
+            >
+                <div className="finish-later-modal-header">
+                    <FormattedMessage
+                        id={'becomeAScratcher.buttons.finishLater'}
+                    />
+                </div>
+                <div className="finish-later-modal-content">
+                    <h3>
+                        <FormattedMessage
+                            id={'becomeAScratcher.finishLater.header'}
+                        />
+                    </h3>
+                    <div>
+                        <FormattedMessage
+                            id={'becomeAScratcher.finishLater.body'}
+                        />
+                        <br />
+                        <FormattedMessage
+                            id={'becomeAScratcher.finishLater.clickBecomeAScratcher'}
+                        />
+                    </div>
+                    <img
+                        className="profile-page-image"
+                        src="/images/onboarding/profile-page-become-a-scratcher-button.svg"
+                    />
+                    <a href={`/users/${user.username}`}>
+                        <Button>
+                            <FormattedMessage
+                                id={'becomeAScratcher.buttons.backToProfile'}
+                            />
+                        </Button>
+                    </a>
+                </div>
+            </Modal>
+            <span>
+                <span className="section">{sectionText}</span>
+            </span>
+            <Button
+                onClick={() => setShowModal(true)}
+                className={`finish-later ${secondary ? 'secondary-finish-later' : ''}`}
+            >
+                <FormattedMessage
+                    id={'becomeAScratcher.buttons.finishLater'}
+                />
+            </Button>
+        </div>
+    );
+};
+OnboardingHeader.propTypes = {
+    user: PropTypes.shape({
+        id: PropTypes.number,
+        thumbnailUrl: PropTypes.string,
+        username: PropTypes.string
+    }),
+    sectionText: PropTypes.string,
+    secondary: PropTypes.bool
+};
+
+const OnboardingNavigation = ({currentPage, totalDots, onNextPage, onBackPage, nextButtonText}) => {
+    const dots = [];
+
+    if (currentPage && totalDots){
+        for (let i = 0; i < totalDots - 1; i++){
```

Comment:
```suggestion
        for (let i = 0; i < totalDots; i++){
```
I think there's an off-by-one error here (it's showing one too few dots)